RELEASE.rst

```diff
@@ -0,0 +1,7 @@
+RELEASE_TYPE: patch
+
+This release will fix some extremely specific circumstances that probably have
```

Comment:
No future tense - "this release fixes ..." by the time anyone reads it!

(I generally describe new of changed behaviour in present tense, and fixed bugs in past tense)